test/serial/irqbalance.bats

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bats
 # vim: set syntax=sh:
 
-load helpers
+load ../helpers
```

Comment:
another option if this doesn't work: you could call `skip` in the `setup()` if an env variable isn't present and then set the env variable on the second bats call

Reply:
kind of like `TEST_USERNS` handling

Reply:
> another option if this doesn't work: you could call `skip` in the `setup()` if an env variable isn't present and then set the env variable on the second bats call

it works I tested it yesterday